pyproject.toml

```diff
@@ -10,5 +10,51 @@ version_scheme =  "post-release"
 local_scheme = "no-local-version"
 write_to =  "stravalib/_version_generated.py"
 
+[project]
+name = "stravalib"
+description = "A Python package that makes it easy to access and download data from the Strava V3 REST API."
+keywords = ["strava", "running", "cycling", "athletes"]
+readme = "README.md"
+dynamic =["version"]
+license = {text = "Apache 2.0 License"} 
+authors = [{name="Hans Lellelid", email="hans@xmpl.org"}]
+maintainers = [
+     {name = "Leah Wasser", email = "leah@pyopensci.org"},
```

Comment:
If you look at the maintainers on test pypi you'll see if you hover over the names you get a weird mailto: string and also 2 names are missing (they are what appear in the hover) super weird. but sourmesh has the same issue.